test/hotspot/jtreg/vmTestbase/nsk/share/jpda/DebugeeProcess.java

```diff
@@ -41,7 +41,7 @@
  * This is an abstract class that declares abstract methods to control
  * debugee VM process.
  * Derived classes should implement these methods corresponding to the mode
- * that the process should be started in (locally, remotely or manually).
+ * that the process should be started in (locally).
```

Comment:
I think this can be dropped

Reply:
Which part?

Reply:
I guess, Alex suggested to simplify this statement, or maybe to remove it.
This part looks a little bit strange when there is only one (local) mode:
 `these methods corresponding to the mode`.

Reply:
There are a lot of references to "local" in comments and API names (in most of the files changed, not just DebugeeProcessl.java). I started down the path of cleaning these all up, but there ended up being a lot, and it distracts from the main purpose of this CR, which is removing BindServer.java. I'd rather do the other "local" cleanups in another PR.